skopt/dummy_opt.py

```diff
@@ -15,9 +15,17 @@ def dummy_minimize(func, bounds, maxiter=1000, random_state=None):
         Function to minimize. Should take a array of parameters and
         return the function values.
 
-    * `bounds` [array-like, shape=(n_parameters, 2)]:
-        - ``bounds[i][0]`` should give the lower bound of each parameter and
-        - ``bounds[i][1]`` should give the upper bound of each parameter.
+    * `dimensions` [list, shape=(n_dims,)]:
+        List of search space dimensions.
+        Each search dimension can be defined either as
+
+        - a `(upper_bound, lower_bound)` tuple (for `Real` or `Integer`
+          dimensions),
+        - a `(upper_bound, lower_bound, "prior")` tuple (for `Real`
+          dimensions),
+        - an instance of a `Dimension` object (for `Real`, `Integer` or
```

Comment:
I'd swap lines 26 and 28 and change to "an instance of a `Dimension` object (`Real`, `Integer` or `Categorical`)" (remove the for)